sql/core/src/main/scala/org/apache/spark/sql/streaming/StreamingQueryManager.scala

```diff
@@ -55,6 +56,11 @@ class StreamingQueryManager private[sql] (sparkSession: SparkSession) extends Lo
   @GuardedBy("awaitTerminationLock")
   private var lastTerminatedQuery: StreamingQuery = null
 
+  sparkSession.sparkContext.conf.get(STREAMING_QUERY_LISTENERS).foreach { classNames =>
+    Utils.loadExtensions(classOf[StreamingQueryListener], classNames,
+      sparkSession.sparkContext.conf).foreach(addListener)
+  }
+
```

Comment:
two comments here: 
1. we need to log the registration here 
2. we need to use try catch for this, it is possible that register fail. this would break the job.

Reply:
Good point. Addressed, please check.